### DISCUSSION TITLE
No documentation about metadata

### DISCUSSION THREAD

**Discussion by noraj** [projectdiscovery/nuclei-templates#6481] [Q&A] [answered]
I saw under `metadata:` that we can put shodan dork under `shodan-query:`. But I wonder is there are more dark tags available and what are they used for.

I didn't find any explanation on https://github.com/projectdiscovery/nuclei-templates/wiki or https://nuclei.projectdiscovery.io/templating-guide/ outside https://nuclei.projectdiscovery.io/nuclei/get-started/#uncover-integration.

As far as I understand is to automate search with uncover.

Maybe there is a `google-query` or `google-dork` or `censys-query` etc. Is the rule `<ENGINE>-query` for the key and the query for the value like `<-uq:>:<-ue>`.

**Comment by noraj** [comment, marked as answer]
I PRed https://github.com/projectdiscovery/nuclei-docs/pull/107

**Comment by ehsandeep** [comment]
Thank you @noraj for updating the docs.
